sygra/core/graph/langgraph/langgraph_factory.py

```diff
@@ -144,14 +144,20 @@ def convert_to_chat_format(self, msgs: list):
         """
         return utils.convert_messages_from_langchain_to_chat_format(msgs)
 
-    def get_test_message(self):
+    def get_test_message(self, is_multi_modal=True):
```

Comment:
It's a small thing but since the default behaviour we would be assuming would be non multimodal so shouldn't it be 

```suggestion
    def get_test_message(self, is_multi_modal=False):
```